### PR TITLE
docs: Add events related doc

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

For https://github.com/appium/appium/issues/13568

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by KazuCocoa** [comment]
@jlipps 
Could you take a look at this when you have a change?
About event APIs

File: docs/en/advanced-concepts/event-timings.md [hunk 1]
```diff
@@ -55,3 +55,8 @@ right before quitting it.
 The Appium team maintains an event timings parser tool that can be used to
 generate various kinds of reports from event timings output:
 [appium/appium-event-parser](https://github.com/appium/appium-event-parser).
+
+## Add a custom event
+
+Since Appium 1.16.0, you can add a custom event.
+You can send a custom event name to Appium server as [Log Event API](docs/commands/session/events/log-event), then Appium server store the time. [Get events](docs/commands/session/events/get-events) returns stored events.
```
Comment (jlipps) [inline docs/en/advanced-concepts/event-timings.md]: > You can send a custom event name to the Appium server using the Log Event API, and the server will store the timestamp. The Get Events command can be used to retrieve named events' timestamps later on.

**Review by jlipps — APPROVED** [review]
amazing, thank you!

**Comment by KazuCocoa** [comment]
thanks :)

**Pull request merged**